src/pymax/interfaces.py

```
@@ -255,3 +162,391 @@ async def start(self) -> None:
+                len(self.dialogs),
+                len(self.chats),
+                len(self.channels),
+            )
+
+        except Exception as e:
+            self.logger.exception("Sync failed")
+            self.is_connected = False
+            if self._ws:
+                await self._ws.close()
+            self._ws = None
+            raise e
```

Ruff (0.14.10)

Use `raise` without specifying exception name

Remove exception name

(TRY201)